README.md

```diff
@@ -1,48 +1,59 @@
-# transfers-indexer
+# Transfers Indexer
 
-This tool allows everyone to save the transfers of MAS made from an EOA forever using the Massa API and a MySQL database.
+The Transfers Indexer is a powerful tool designed for archiving MAS token transfers from Externally Owned Accounts (EOA) on the Massa blockchain. It utilizes the API exposed by the [Massa node](https://github.com/massalabs/massa) for data collection and stores this information in a MySQL database.
 
-## Setup
+## Prerequisite
 
-### Database
+- **Massa Node with Execution Trace**: You must have access to a Massa node compiled with the `execution-trace` feature or run the source code with `cargo run -r --features execution-trace`.
 
-You need to have a MySQL backend running to use this program. You can either use your own or, if you don't have one, we added a `docker-compose.yml` that run a MySQL in this repository.
+## Setup Instructions
 
-When you have your database backend running make sure to configure the `.env` file at the root of this repository to match the credentials, host, etc... for your database.
+To start the indexer, execute:
 
-### Massa node
-
-You need to have a massa-node connected to this project in order to gather the transfers from the network.
-This node needs to be launched using the feature `execution-trace`. The command to launch the node with the feature is : 
+```sh
+cargo run -r
```

Comment:
For me it's less clear now as people will not link their database before launching the indexer and they don't know that they need to launch the database backend before. I prefer explaining the launching of the MySQL backend before

Reply:
Is it better now?

Reply:
yes